zookeeper-client/zookeeper-client-c/libzookeeper.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/lib
+includedir=${prefix}/include/
+
+Name: @CMAKE_PROJECT_NAME@
+Description: Zookeeper C client library
+Version: @CMAKE_PROJECT_VERSION@
+Libs: -L${libdir} -lzookeeper
+Libs.private: @LIBS@ ${libdir}/libzookeeper@CMAKE_STATIC_LIBRARY_SUFFIX@
```

Comment:
Why is `libzookeeper.a` in `Libs.private`?  It seems `-lzookeeper` in `Libs:` should be enough.